designer/src/test/java/org/restcomm/connect/rvd/logging/CustomLoggerTest.java

```diff
@@ -92,9 +92,9 @@ public void test1MBSize() throws IOException {
         }
         String content = FileUtils.readFileToString(new File(basepath + ".log"));
         Assert.assertTrue(content.contains("9999 - A really"));
-        Assert.assertTrue(content.contains("6337 - A really"));
+        Assert.assertTrue(content.contains("6377 - A really"));
         content = FileUtils.readFileToString(new File(basepath + "-1.log"));
-        Assert.assertTrue(content.contains("6336 - A really"));
+        Assert.assertTrue(content.contains("6376 - A really"));
```

Comment:
Test still breaks here in my environment. Don't waste any time on this.

Reply:
works here sorry

Reply:
maybe we could relax the assertions to make it work everywhere,or simply ignore the test

Reply:
:+1: relaxed assertions